apps/wallet/src/ui/app/shared/Link.tsx

```diff
@@ -9,13 +9,17 @@ import { ButtonOrLink, type ButtonOrLinkProps } from './utils/ButtonOrLink';
 const styles = cva(
     [
         'transition flex flex-nowrap items-center justify-center outline-none gap-1 w-full',
-        'no-underline bg-transparent p-0 border-none',
+        'bg-transparent p-0 border-none',
         'active:opacity-70',
         'disabled:opacity-40',
         'cursor-pointer group',
     ],
     {
         variants: {
+            underline: {
```

Comment:
do we want this behavior on all of our links? ideally the underline/hover behavior would be consistent throughout

Reply:
@mystie711 can provide some more context here, but I think there are some scenarios where the underline behavior should change:

- Text next to a link that's all the same color -> underline by default, no underline on hover
- Text next to a link with differing colors -> no underline by default, underline on hover 

I'll defer to him on the answer to your Q 😄